colossalai/booster/plugin/low_level_zero_plugin.py

```diff
@@ -318,6 +381,12 @@ def configure(
         dataloader: Optional[DataLoader] = None,
         lr_scheduler: Optional[LRScheduler] = None,
     ) -> Tuple[nn.Module, OptimizerWrapper, Callable, DataLoader, LRScheduler]:
+        if self.lora_enabled:
+            from peft import PeftModel
+            assert isinstance(model, PeftModel), "The model should have been wrapped as a PeftModel when self.lora_enabled is True"
+            self.add_lora_params_to_optimizer(model, optimizer)
```

Comment:
Optimizer may by None and this function call may lead to error

Reply:
updated.